mpas_analysis/shared/io/namelist_streams_interface.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/python
+"""
+    Module of classes / routines to manipulate fortran namelist and streams
+    files.
+
+    Phillip Wolfram
+    LANL
+    04/24/2014
+"""
+
+from subprocess import check_output, call
+from lxml import etree
+
+class NameList:
+    """
+    Class for fortran manipulation of namelist files, provides
+    read and write functionality
+    """
+
+    # constructor
+    def __init__(self, fname, readonly=True):
+        # set readonly bit
+        self.readonly = readonly
+        # input file name
+        self.fname = fname
+
+    def read(self, name):
+        # shell return value
+        return_val = check_output(['awk', '/'+name+'/{printf $3}', self.fname])
+        return_val.rstrip('\n')
+        try:
+            return_val = float(return_val)
+        except:
+            return_val = return_val.strip('"').strip("'")
+        return return_val
+
+    def write(self, name, value):
```

Comment:
I don't think we want namelist write functionality in this repo.  @pwolfram, can you give me an example of where we might need this in this repo?